backend/src/main/java/de/laura/project/model/TriviaPointCategoryDTO.java

```diff
@@ -0,0 +1,20 @@
+package de.laura.project.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+
+public class TriviaPointCategoryDTO {
+
+    private String category;
+    private List<TriviaPointDetails> triviaPointDetails;
+
+}
```

Comment:
Seems the TriviaPointCategory + ""DTO has the same structure? May you need only one of them?

Reply:
In one category is a String in the other an int